src/test/java/io/tarantool/driver/integration/ProxyTarantoolClientExampleIT.java

```diff
@@ -0,0 +1,109 @@
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Ivan Dneprov
+ * <p>
+ * WARNING: If you updated the code in this file, don't forget to update the docs/ProxyTarantoolClient.md permalinks!
+ */
+public class ProxyTarantoolClientExampleIT extends SharedCartridgeContainer {
+
+    private static TarantoolClient<TarantoolTuple, TarantoolResult<TarantoolTuple>> client;
+
+    public static String USER_NAME;
+    public static String PASSWORD;
+
+    private static final String SPACE_NAME = "test__profile";
+    private static final String PK_FIELD_NAME = "profile_id";
+
+    @BeforeAll
+    public static void setUp() throws Exception {
+        startCluster();
+        initClient();
+        truncateSpace(SPACE_NAME);
+    }
+
+    public static void initClient() {
+        client = TarantoolClientFactory.createClient()
+            // If any addresses or an address provider are not specified,
+            // the default host 127.0.0.1 and port 3301 are used
+            .withAddress(container.getHost(), container.getPort())
+            // For connecting to a Cartridge application, use the value of cluster_cookie parameter in the init.lua file
+            .withCredentials(container.getUsername(), container.getPassword())
+            // Specify using the default CRUD proxy operations mapping configuration
+            .withProxyMethodMapping()
+            // You may also specify more client settings, such as:
+            // timeouts, number of connections, custom MessagePack entities to Java objects mapping, etc.
+            .build();
+    }
+
+    private static void truncateSpace(String spaceName) {
+        client.space(spaceName).truncate().join();
+    }
+
+    @Test
+    public void clusterInsertSelectDeleteTest() throws ExecutionException, InterruptedException {
+        TarantoolSpaceOperations<TarantoolTuple, TarantoolResult<TarantoolTuple>> space = client.space(SPACE_NAME);
+
+        // Use TarantoolTupleFactory for instantiating new tuples
+        TarantoolTupleFactory tupleFactory = new DefaultTarantoolTupleFactory(
+            client.getConfig().getMessagePackMapper());
+
+        TarantoolTuple tuple;
+        List<TarantoolTuple> tarantoolTuples = new ArrayList<>();
+        for (int i = 0; i < 20; i++) {
+            // Pass the field corresponding to bucket_id as null for tarantool/crud to compute it automatically
+            tarantoolTuples.add(tupleFactory.create(1_000_000 + i, null, "FIO", 50 + i, 100 + i));
+        }
+        space.insertMany(tarantoolTuples).get();
+
+        // Primary index key value will be determined from the tuple
+        tuple = tupleFactory.create(1_000_000 - 1, null, "FIO", 50 - 1, 100 - 1);
+        Conditions conditions = Conditions.after(tuple);
+
+        TarantoolResult<TarantoolTuple> selectResult = space.select(conditions).get();
```

Comment:
nit: I don't know if it is more beautiful or not?
```suggestion
        TarantoolResult<TarantoolTuple> selectResult = space.select(
            Conditions.greaterOrEquals(PK_FIELD_NAME, 1_000_000)
        ).get();
```

Reply:
I've tried to show multiple filter options